packages/e2e.generated.api-namespace.local/src/generatedNoCheck/ontology/queries/getTask.ts

```diff
@@ -46,6 +47,7 @@ export type getTask = getTask.Signature;
 
 export const getTask: getTask.Definition = {
```

Comment:
It seems that we still use the old .Definition instead of the new style, will address in a FLUP PR